index.php

```diff
@@ -9,76 +41,46 @@
 </head>
 
 <body>
-    <h1>Portfolio Edit Page</h1>
     <div>
-        <h2>Header Nav Bar Links</h2>
+    <h2>About Me</h2>
+    <h2>Edit</h2>
+        <form id="editAboutMe" action="index.php" method="post">
```

Comment:
This is the form you could have returned in the function I mentioned above...

So instead of `<form id="editAboutMe" action="index.php" method="post">`etc with `<?php echo listingSubtitles($subtitleArray) ?>` inside the select, you could just be running the function that would return this entire form...

Reply:
Thank you, as above.